.circleci/config.yml

```diff
@@ -1,11 +1,32 @@
----
-workspace_root: &workspace_root
-  /go/src/github.com/hashicorp/terraform-aws-nomad
-
 defaults: &defaults
-  working_directory: *workspace_root
-  docker:
-    - image: 087285199408.dkr.ecr.us-east-1.amazonaws.com/circle-ci-test-image-base:latest
+  # docker:
+  #   - image: 087285199408.dkr.ecr.us-east-1.amazonaws.com/circle-ci-test-image-base:latest
+  machine:
+    image: circleci/classic:201711-01    
```

Comment:
We used to test in Docker... Is there a reason to move to machine for this repo?

Reply:
I was getting various permission denied errors when trying to `install_gruntwork_utils`. It seemed the image we were using before was locked down such that it didn't have permission to delete from `/tmp`

Reply:
Why haven't we seen that error elsewhere or before? We use that image in a few places, right?

Reply:
I'm not sure what this error is. I've been using the machine image for my other tests. So I just grabbed a known working image.

What's the advantage of using Circle's docker image vs Circle's Machine image?

Reply:
Build speed. The Docker image fires up quite a bit faster than the machine image. Also, Docker image can have many of our utilities/dependencies pre-installed, so you don't have to install them from scratch every single build. This can shave several minutes on each run, which is a big deal.

The _only_ reason (usually) not to use a Docker image is if you're testing something that itself needs to build or run Docker images or Docker Compose. Running Docker in Docker is a bit of a pain, so there, we use machine images.